setup.cfg

```diff
@@ -57,11 +57,14 @@ test =
     pytest-cov
     %(style)s
     %(nk)s
+logger = 
```

Comment:
In this way, logging through loguru is not mandatory - it's ok to do so (in fact, might be better to certain effects), but then all loguru imports should be enclosed in `try..except` statements, all within the functions that need them, and since some sort of logging is important, you should offer alternatives.

I'm happy to chat about this if you want!